configs/Kconfig

```diff
@@ -71,6 +71,10 @@ config LOADER_GIF
     bool "Enable GIF loader"
     default y
 
+config LOADER_TinyVG
```

Comment:
I checked the [specification](https://tinyvg.tech/download/specification.pdf), and there seems to be no official abbreviation of TinyVG.

Maybe we should use `LOADER_TVG` for consistency.